backend/src/main/java/com/github/nickklock/backend/secruity/SecurityConfig.java

```diff
@@ -35,12 +36,15 @@ public SecurityFilterChain filterChain(HttpSecurity httpSecurity) throws Excepti
                 .csrf(csrf -> csrf
                         .csrfTokenRepository(tokenRepository)
                         .csrfTokenRequestHandler(requestHandler))
+                .sessionManagement(session ->
+                        session.sessionCreationPolicy(SessionCreationPolicy.ALWAYS))
                 .httpBasic()
                 .authenticationEntryPoint((request, response, authException) ->
                         response.sendError(HttpStatus.UNAUTHORIZED.value(), HttpStatus.UNAUTHORIZED.getReasonPhrase()))
                 .and()
                 .authorizeHttpRequests(auth -> auth
                         .requestMatchers("/api/users/me").permitAll()
+                        .requestMatchers("/api/users/login").permitAll()
```

Comment:
Nice job working with the allow list! Keep in mind if you deploy only the spring application with your react bundled into it, you will need to find a way to allow all the frontendstuff. But perfect approach 👍